examples/nl-core-FLACCpainScale-01.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-FLACCpainScale-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-FLACCpainScale"/>
+   </meta>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="108291000146105"/>
+         <display value="FLACC-pijnmeetinstrument"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <effectiveDateTime value="2016-12-21T12:05:00+01:00"/>
+   <valueInteger value="6"/>
+   <note>
+      <text value="Controle voor toediening pijnmedicatie."/>
+   </note>
+   <component>
+      <code>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1"/>
+            <code value="12013003"/>
+            <display value="FLACCpijnScore Gezicht"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.21.1"/>
```

Comment:
The urn oid code is 2.16.840.1.113883.2.4.3.11.60.40.2.12.13.1 in the zib? Where does this value come from?